### PR TITLE
Maintenance

### PR DESCRIPTION
**Pull request by HolQue**

### CODE REVIEW

File: 100_variables_and_datatypes/doc_src/tutorial-01.robot.rst [hunk 1]
```diff
@@ -183,7 +183,7 @@ With these variables test case ``01_02`` does the following arithmetical computa
 
 **Outcome**
 
-Where necessary the RobotFramework AIO automatically converts the data types to enable arithmetic computations and catenations.
+Where necessary the Robot Framework automatically converts the data types to enable arithmetic computations and catenations.
```
Comment (test-fullautomation) [inline 100_variables_and_datatypes/doc_src/tutorial-01.robot.rst:186]: Hi Holger,
now I'm confused.

Our tutorial is specific for  RobotFramework AIO. Therefore it makes sense to refer to it.
In this case here of course there is no contribution of RobotFramework AIO, this is Robot Framework Core functionality, thereore Robot Framework is correct.

But if meaningful RobotFramework AIO can be referenced in this tutorial, too.

Thank you,
 Thomas

File: README.rst [hunk 2]
```diff
@@ -18,17 +18,18 @@ Package Description
 Getting Started
 ---------------
 
-The RobotFramework AIO tutorial contains robot code examples together with the documentation explaining how to use these
+This Robot Framework tutorial contains robot code examples together with the documentation explaining how to use these
```
Comment (test-fullautomation) [inline README.rst:21]: Here we would have ``RobotFramework AIO`` tutorial. This is what is is.

**Review by test-fullautomation — CHANGES_REQUESTED** [review]
Hi Holger,

looks good to me!
Please check my findings.

Thank you,
 Thomas

File: README.rst (hunk 2 above)
Comment (HolQue) [reply README.rst:21]: Hi Thomas,

as per PyPi stand-alone philosophy of the described components I removed all references to "AIO".

Comment (test-fullautomation) [reply README.rst:21]: Hi Holger,
very good!
Thank you,
Thomas

**Review by test-fullautomation — APPROVED** [review]
Hi Holger,
looks good to me!
Thank you,
 Thomas

**Pull request merged**